documentation/modules/ROOT/pages/integration-with-aap.adoc

```diff
@@ -0,0 +1,176 @@
+
+[#acm-hooks-toward-aap]
+=== ACM Hooks Toward AAP
+
+This section will provide a high level overview of a governance-based ACM hook toward AAP automation. This process involves the Policy, PlacementBinding, and PlacementRule CRDs discussed in link:rhacm-policies.html#policies[RHACM Policies], as well as the CRDs PolicyAutomation and AnsibleJob.
+
+First, a Policy is created detailing the necessary state of the cluster after AAP automation is complete, along with a PlacementRule defining which clusters this policy applies to and a PlacementBinding the Policy to the PlacementRule. This governance structure ensures that a cluster will not be compliant until our automation has successfully run and the desired end state of imperative configuration is met. 
+
+Next, we create our PolicyAutomation in the form of an AnsibleJob, binding it to our defined Policy. Upon our Policy binding to a spoke cluster, RHACM will trigger our PolicyAutomation defined AnsibleJob, which starts an automation job on the Automation Controller, passing a list of targeted clusters (the clusters PlacementRule has bound to) as an extra variable. 
+
+Now, the same Policy that triggered the Ansible job also checks for the necessary cluster state seeking to be achieved by the completion of the job. Only when the job completes successfully and the cluster reaches a compliant state does the automation policy reach a compliant status.
+
+[#workflow-overview]
+== Workflow Overview
+
+Now that we are familiar with AAP in OpenShift and the necessary components for integration, let's walk through an Ansible integration workflow.
+
+[#general-workflow]
+=== General AAP Integration Workflow
+
+At a high level, the steps to AAP integration with 5G RAN ZTP deployments are as follows:
+
+* Mirror AAP operator to disconnected registry (outside the scope of this lab, see the _Ansible Automation Platform mirroring_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog])
+* Install AAP operator, deploy AutomationHub and AutomationController (already running on OpenShift cluster, see the _Ansible Automation Platform operator installation_ section of this link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[blog] and the CRD examples for link:integration-with-aap.html#automation-hub[AutomationHub] and link:integration-with-aap.html#automation-controller[AutomationController])
+* Upload necessary Ansible collections to Automation Hub (already uploaded, see uploaded collections at automation-hub-aap.apps.hub.5g-deployment.lab and instructions for this process in the link:lab-environment.html#upload-collections-to-aap-hub[Lab Environment] section)
```

Comment:
We will do this in our automation. Pointing users to Lab Environment docs is not advised, it can be confusing (they're not supposed to run these when they get an env from demo.redhat.com)